cmd/serverless/main.go

```diff
@@ -197,24 +192,25 @@ func runAgent(stopCh chan struct{}) (daemon *serverless.Daemon, err error) {
 		log.Warn("An API Key has been set in multiple places:", strings.Join(apikeySetIn, ", "))
 	}
 
-	// try to read apikey from KMS
+	// try to read API key from KMS
 
 	var apiKey string
 	if apiKey, err = readAPIKeyFromKMS(); err != nil {
 		log.Errorf("Error while trying to read an API Key from KMS: %s", err)
 	} else if apiKey != "" {
 		log.Info("Using deciphered KMS API Key.")
-		os.Setenv(apiKeyEnvVar, apiKey) // it will be catched up by config.Load()
+		log.Debug(apiKey)
```

Comment:
This is the plainText API right ? Do we really want to output it ? It may be a security risk ?

Reply:
Good catch, this was for my own debugging! Will remove. Although, worth noting that CloudWatch will automatically mask it.